src/gui/CMakeLists.txt

```diff
@@ -9,5 +9,6 @@ set(gui_SRCS
 	${CMAKE_CURRENT_SOURCE_DIR}/guiTable.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/guiVolumeChange.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/intlGUIEditBox.cpp
+	${CMAKE_CURRENT_SOURCE_DIR}/profilergraph.cpp
```

Comment:
Also add the file to our Android build source list: `build/android/jni/Android.mk`

Reply:
done